tests/x11regressions/empathy/empathy_irc.pm

```diff
@@ -48,16 +48,13 @@ sub run() {
     };
 
     # join openqa channel
-    if (sle_version_at_least('12-SP2')) {
+    send_key "ctrl-j";
+    if (!check_screen 'empathy-join-room', 4) {
```

Comment:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/46f52261a71dec77db5acb9378e667a5ac9b3f2a/tests/x11/firefox.pm#L53 might be a good example how to check for multiple tags within one assert_screen call. The file also has other examples where `check_screen(…, 0)` makes sense